src/components/MessagesRow.tsx

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import {StyleProp, View, ViewStyle} from 'react-native';
+import useLocalize from '@hooks/useLocalize';
+import useThemeStyles from '@styles/useThemeStyles';
+import CONST from '@src/CONST';
+import DotIndicatorMessage from './DotIndicatorMessage';
+import Icon from './Icon';
+import * as Expensicons from './Icon/Expensicons';
+import PressableWithoutFeedback from './Pressable/PressableWithoutFeedback';
+import Tooltip from './Tooltip';
+
+// const propTypes = {
+//     /** The messages to display */
+//     messages: PropTypes.objectOf(
+//         PropTypes.oneOfType([PropTypes.oneOfType([PropTypes.string, PropTypes.object]), PropTypes.arrayOf(PropTypes.oneOfType([PropTypes.string, PropTypes.object]))]),
+//     ),
+
+//     /** The type of message, 'error' shows a red dot, 'success' shows a green dot */
+//     type: PropTypes.oneOf(['error', 'success']).isRequired,
+
+//     /** A function to run when the X button next to the message is clicked */
+//     onClose: PropTypes.func,
+
+//     /** Additional style object for the container */
+//     containerStyles: stylePropTypes,
+
+//     /** Whether we can dismiss the messages */
+//     canDismiss: PropTypes.bool,
+// };
+
+// const defaultProps = {
+//     messages: {},
+//     onClose: () => {},
+//     containerStyles: [],
+//     canDismiss: true,
+// };
+
+type MessagesRowProps = {
+    messages: Record<string, string>;
+    type: 'error' | 'success';
+    onClose: () => void;
+    containerStyles: StyleProp<ViewStyle>;
+    canDismiss: boolean;
```

Comment:
Also please, add comments to type props
```suggestion
    onClose?: () => void;
    containerStyles?: StyleProp<ViewStyle>;
    canDismiss?: boolean;
```

Reply:
I don't know what is happening in my head but I am keep forgetting about that 😅